### ISSUE TITLE
Awkward linebreak between "Mozilla" and "VPN", in "You are up to date" dialog

### ISSUE THREAD

**Issue by dholbert** [mozilla-mobile/mozilla-vpn-client#2677]
STR:
1. Tap the "cog" to go to settings
2. Tap "About us"
3. Tap "Check for Updates"

ACTUAL RESULTS:
The dialog pops up and looks like this, with the text *almost but not quite* fitting on one line, with "VPN" wrapped to its own centered line:
```
You currently have the latest version of Mozilla
                     VPN

                   [Okay]
```

EXPECTED RESULTS:
Assuming that we have to linewrap here, this would probably look a little less awkward if "Mozilla VPN" were wrapped as a unit, e.g.:
```
   You currently have the latest version of
                  Mozilla VPN

                   [Okay]
```

...which we could achieve e.g. by using a non-breaking space in that part of the string.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-1687)

**Comment by dholbert** [comment]
Version/device info:
I'm using a Google Pixel 4a with Android 12 and Mozilla VPN version 2.7.0

**Comment by dholbert** [comment]
![Screenshot_20220204-122457](https://user-images.githubusercontent.com/426803/152599072-46a2a053-27f0-4d58-991d-f6a43e9a91c9.png)

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified as fixed on the last VPN 2.8 build on both Windows and Android 10.

Postfix screenshot below.

!image-20220317-153252.png|width=459,height=708!
